core/controllers/acl_decorators.py

```diff
@@ -204,7 +205,7 @@ def test_can_view(self, comma_separated_skill_ids, **kwargs):
         try:
             for skill_id in skill_ids:
                 skill_domain.Skill.require_valid_skill_id(skill_id)
-        except Exception:
+        except utils.ValidationError:
```

Comment:
While I agree in principle having more specific errors, in this particular context the additional robustness of handling unknown errors may be important. What happens if a non-validation error is raised here or in the other decorators? I'm guessing the user experience won't be different, just the backtraces. Does that align with your thinking?

Reply:
Yep the experience is the same, I have tested this.

Reply:
Thanks for checking.